openapi/multi-yaml/index.yaml

```diff
@@ -79,223 +79,141 @@ tags:
-            "signature": "4bHXoVaoR",
-            "account_id": "acc_456subaccount",
-            "status": "valid",
-            "card": {
-              "id": "pm_autoclonedxyz789",
-              "name": "Lindsay Whalen",
-              "acct_last_four": "4242",
-              "brand": "visa",
-              "token": "pm_autoclonedxyz789",
-              "month": "5",
-              "year": "2025"
-            }
-          }
-        }
-      }
+      });
+
+      // Example: Add payment methods to group
+      const updatedGroup = await fetch(`https://api.justifi.ai/v1/payment_method_groups/${groupId}`, {
+        method: 'PATCH',
+        headers: {
+          'Authorization': `Bearer ${token}`,
+          'Sub-Account': platformWalletAccountId,  // Platform wallet account
+          'Content-Type': 'application/json'
+        },
+        body: JSON.stringify({
+          "payment_method_ids": ["pm_walletaccxyz", "pm_walletaccabc"]
+        })
+      });
       ```
 
-      ### [Checkout Flow](http://localhost:3001/api-spec#tag/Checkouts/operation/CreateCheckout) Example
-
-      When using platform wallet payment methods with checkouts:
-      1. Create a checkout for the sub-account, specifying a payment method group from the wallet account
-      2. Complete the checkout using a payment method token from the wallet account
-
-      ##### Example Request: Create Checkout
-      ```json
-      {
-        "amount": 10000,
-        "description": "order_xyz",
-        "origin_url": "http://localhost:3000",
-        "payment_method_group_id": "pmg_xyz123",
-        "statement_descriptor": "Big Business"
-      }
-      ```
+      ### 2. Process Payments
+      You can process payments using wallet payment methods through either our component or API approach. All payment operations require the processing sub account ID in the Sub-Account header.
 
-      ##### Example Response: Checkout Created
-      ```json
-      {
-        "id": "cho_123xyz",
-        "type": "checkout",
-        "page_info": null,
-        "data": {
-          "id": "cho_123xyz",
-          "payment_intent_id": null,
-          "account_id": "acc_subacc123",
-          "platform_account_id": "acc_plat123",
-          "payment_amount": 10000,
-          "payment_currency": "USD",
-          "payment_description": "",
-          "payment_methods": [],
-          "payment_method_group_id": "pmg_xyz123",
-          "status": "created",
-          "mode": "test",
-          "payment_settings": {
-            "credit_card_payments": true,
-            "ach_payments": true,
-            "bnpl_payments": false,
-            "insurance_payments": false
-          },
-          "successful_payment_id": null,
-          "statement_descriptor": null,
-          "application_fees": null,
-          "created_at": "2024-10-29T21:22:02.857Z",
-          "updated_at": "2024-10-29T21:22:02.857Z",
-          "completions": []
-        }
-      }
+      #### Via Component
```

Comment:
Here I'd say:
' #### Via Checkout component
And then just refer them to the 'Checkout via Component' (https://docs.justifi.tech/api-spec#tag/Checkout-via-Component) walkthrough. Then you won't need to code examples for that approach.